lte/gateway/docker/mme/Dockerfile.rhel8

```diff
@@ -416,7 +416,7 @@ RUN yum install -y python3-pip && \
 # For developer's to have the same run env as in target image to debug
 # Copy the configuration file templates and mean to modify/generate certificates
 WORKDIR /magma-mme/bin
-RUN cp $C_BUILD/oai/oai_mme/mme oai_mme
+RUN cp $C_BUILD/core/oai/oai_mme/mme oai_mme
 RUN cp $C_BUILD/sctpd/sctpd .
```

Comment:
Is the `sctpd` location correct?

Reply:
Nope :D. It needs to be under src. I think I don't know how to build this locally? Will do that and then report back. Thanks for the instructrions.